packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -779,14 +779,15 @@ export default class LocusInfo extends EventsScope {
   /**
    * @param {Object} info
    * @param {Object} self
+   * @param {string} meetingState
    * @returns {undefined}
    * @memberof LocusInfo
    */
-  updateMeetingInfo(info, self) {
+  updateMeetingInfo(info, self, meetingState) {
     if (info && (!isEqual(this.info, info))) {
       const roles = self ? SelfUtils.getRoles(self) : this.parsedLocus.self?.roles || [];
 
-      const parsedInfo = InfoUtils.getInfos(this.parsedLocus.info, info, roles);
+      const parsedInfo = InfoUtils.getInfos(this.parsedLocus.info, info, roles, meetingState === MEETING_STATE.STATES.JOINED);
```

Comment:
suggestion: I think that this comparison is better living in the SelfUtils. Given that meetingState is from parsedLocus.self you can add the necessarily comparison function as a util in SelfUtils. I don't think it is necessary to pass the meetingState in separately as you already have self

Reply:
modified